### DISCUSSION TITLE
How many 🔎🔎 are there?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1083] [Galaxy Trivia] [answered]
Move from letter to letter and collect the necessary letters for FIND (in any order). Count the number of different routes you can take if you always start with the central F and always move either horizontally or vertically.

```
      D
    N I D
  D N D I D
D I N F I I N
  I N D D I
    D I D
      N
```

**Comment by peterbaccaro** [comment]
There are **5** different routes. 

![image](https://github.com/ParanoidUser/yolo/assets/74666079/c06694c4-3b86-4929-b22a-183d8d4899c5)

**Comment by thuongtruong1009** [comment, marked as answer]
Let's count the number of different routes to form the word "FIND" starting from the central 'F' and moving either horizontally or vertically.

To form the word "FIND," we need to collect the letters 'F', 'I', 'N', and 'D' from the grid. Here's one possible path to collect these letters:

```
   D
   I
 D N D
D I N F I I N
 I N D D I
   D I
     N
```

To count the total number of different routes, we can explore all the possible paths from the central 'F' while following the rule of moving either horizontally or vertically. However, manually counting all the paths can be quite tedious due to the complexity of the grid.

To make the counting process easier, we can use combinatorics. Since we need to collect four letters (F, I, N, D), and each letter can be collected multiple times in the grid, this problem can be framed as a combination problem.

The number of different routes to collect all four letters is equal to the number of ways we can choose 4 out of the total number of occurrences of F, I, N, and D in the grid.

Let's count the occurrences of each letter in the grid:

- F: 1 occurrence
- I: 6 occurrences
- N: 4 occurrences
- D: 4 occurrences

Now, we need to calculate the number of ways to choose 4 out of these occurrences. This can be done using the formula for combinations:

nCr = n! / (r!(n-r)!),

where n is the total number of occurrences of the letters (in this case, n = 1 + 6 + 4 + 4 = 15), and r is the number of letters we want to choose (r = 4).

So, the number of different routes to collect the letters "FIND" is:

15C4 = 15! / (4! * (15-4)!) = 1365.

There are a total of 1365 different routes to form the word "FIND" starting from the central 'F' and moving either horizontally or vertically.
